dissect/target/plugins/filesystem/ntfs/mft.py

```diff
@@ -133,12 +137,15 @@ def check_compatible(self) -> None:
         ]
     )
     @arg("--compact", action="store_true", help="compacts the MFT entry timestamps into a single record")
+    @arg("--fs", type=int, default=None, help="optional filesystem index, zero indexed")
+    @arg("--start", type=int, default=0, help="the first MFT segment number")
+    @arg("--end", type=int, default=-1, help="the last MFT segment number")
     @arg(
         "--macb",
         action="store_true",
         help="compacts the MFT entry timestamps into aggregated records with MACB bitfield",
     )
-    def mft(self, compact: bool = False, macb: bool = False):
+    def mft(self, compact: bool = False, fs: Optional[int] = None, start: int = 0, end: int = -1, macb: bool = False):
```

Comment:
```suggestion
    def mft(self, compact: bool = False, fs: int | None = None, start: int = 0, end: int = -1, macb: bool = False):
```

Reply:
Fixed

Reply:
Can you also add a `from __future__ import annotations` at the top?

Reply:
The CI fails due to this change https://github.com/fox-it/dissect.target/actions/runs/10266162635/job/28404734201?pr=672#step:10:41

Reply:
The previous change passes the tests.

Reply:
I know, if you add `from __future__ import annotations` at the top of the file it will fix it 😄

Reply:
Nice, I did not know that Python has this feature.